userdatamodel/user.py

```diff
@@ -132,6 +132,18 @@ class UserAuditLog(Base):
     new_values = Column(JSONB, server_default=text("'{}'"))
 
 
+class CertAuditLog(Base):
+    __tablename__ = "cert_audit_logs"
+
+    id = Column(BigInteger, primary_key=True)
+    timestamp = Column(DateTime, server_default=text("now()"), nullable=False)
+    operation = Column(String, nullable=False)
+    user_id = Column(Integer, nullable=False)
+    username = Column(String(255), nullable=False)
```

Comment:
Are we just including `user_id` for our own purposes? Just curious because it's a db pkey but presumably doesn't mean much to an auditor

Reply:
yeah because there is script basically joining the result from `user_audit_logs` and `cert_audit_logs` table by `user_id` to figure out which user has aggreed this ToU/PP

Reply:
Ahh I see. OK thanks!